frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx

```diff
@@ -45,13 +45,20 @@ export const TicketsList = ({ tickets }: TicketsListProps) => {
 	const availableTemplatesIds = uniq(tickets.map(({ type }) => type));
 	const availableTemplates = templates.filter(({ _id }) => availableTemplatesIds.includes(_id));
 	const queries = TicketsCardHooksSelectors.selectFilteringQueries();
+	const selectedPinId = TicketsCardHooksSelectors.selectSelectedTicketPinId();
+	const isShowingPins = TicketsCardHooksSelectors.selectIsShowingPins();
 
 	const filteredByQueriesAndCompleted = TicketsCardHooksSelectors.selectTicketsFilteredByQueriesAndCompleted();
 	const filteredItems = TicketsCardHooksSelectors.selectTicketsWithAllFiltersApplied();
 
 	const toggleTemplate = (templateId: string) => TicketsCardActionsDispatchers.setTemplateFilters(xor(selectedTemplates, [templateId]));
 	const onQueriesChange = (newQueries) => TicketsCardActionsDispatchers.setQueryFilters(newQueries);
 
+	useEffect(() => {
+		if (!selectedPinId) return;
+		ViewerService.setSelectionPin({ id: selectedPinId, isSelected: isShowingPins ?? true });
+	}, [selectedPinId, isShowingPins]);
```

Comment:
When "show pins" is set to `false`, clicking on either the background of the tickets list or a template chip, or typing anything in the search bar, resets that value to `true`

https://github.com/user-attachments/assets/d7c457c8-2c10-41de-bff2-db0c6542da59